src/app/components/Button/Button.tsx

```diff
@@ -0,0 +1,16 @@
+type ButtonProps = {
+	text: string;
+	width: string;
+};
+
+const Button = ({ text, width }: ButtonProps) => {
+	return (
+		<button
+			className={`h-[39px] w-[${width}] rounded-full bg-[#805DBE] text-white text-xs font-bold hover:bg-violet-700`}
```

Comment:
don't use height or width use padding. you should never use height or width.

Reply:
Apologies on that. I removed all props, made it a general button and moved this component within the route folder.